generic-sensor/idlharness.https.window.js

```diff
@@ -0,0 +1,23 @@
+// META: script=/resources/WebIDLParser.js
+// META: script=/resources/idlharness.js
+
+// https://w3c.github.io/sensors/
+
+'use strict';
+
+function cast(i, t) {
+  return Object.assign(Object.create(t.prototype), i);
+}
+
+idl_test(
+  ['generic-sensor'],
```

Comment:
s/generic-sensor/sensors

It's file name under `interfaces/` folder is `sensors.idl`

Reply:
Oops, I didn't see you've rename the file. :p